src/Microsoft.AspNet.Server.Kestrel/Filter/StreamSocketOutput.cs

```diff
@@ -62,6 +95,7 @@ public MemoryPoolIterator2 ProducingStart()
 
         public void ProducingComplete(MemoryPoolIterator2 end)
         {
+            // sync writes with is immedate == true as over Stream
```

Comment:
No :(

Also that its `immedate: true` rather than `immedate:false` as its over `Stream` rather than `ISocketOutput` which has those overloads.